### PR TITLE
Increase contrast between text and background

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description

- Increase color contrast in the header, bridge page, vote page, and delegate page. 

Part of #164

### CODE REVIEW

**Comment by alexkeating** [comment]
Current dependencies on/for this PR:
* main
  * **PR #170** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/170?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/170?utm_source=stack-comment).

**Review by wildmolasses — COMMENTED** [review]
just the one change that i'll push back on a bit, otherwise lgtm

File: components/Header.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ const NavButtons = () => {
     <div>
       <div>
         <nav
-          className="flex space-x-4 my-3 bg-white p-4 bg-opacity-10 rounded-xl sm:my-0"
+          className="flex space-x-4 my-3 bg-white p-4 bg-opacity-60 rounded-xl sm:my-0"
```
[the remark this answers is not in the document]
Comment (alexkeating) [reply components/Header.tsx:43]: Yes, it was tough to feel out. I was using this tool to get a sense if the contrast followed accessibility guidelines. I can play around and see what the range is for making it less opaque

https://coolors.co/contrast-checker/112a46-acc8e5

[the remark this answers is not in the document]
Comment (alexkeating) [reply components/Header.tsx]: https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/170/commits/242d9f960448a640316370235457811b735610e4

**Pull request merged**